web/app/forms.py

```diff
@@ -16,11 +16,44 @@ class SearchPlayerForm(Form):
     submit = SubmitField()
 
 class TournamentForm(Form):
-    event_name = StringField("Tornament Name", validators=[Required()])
-    # start_date = db.Column(db.DateTime)
-    venue = StringField("Venue", validators=[Required()])
+    event_name = StringField("Tournament Name", validators=[Required()])
+
+    venue = StringField("Venue (Optional)") # Venue info Optional
+    venue_address = StringField("Venue Address (Optional)")
+    venue_state = StringField("State (Optional)")
+    venue_zip = StringField("Zip (Optional)")
+
+    start_date = StringField("Start Date", validators=[Required()]) # these will change to some date format
+    end_date = StringField("End Date (Optional)")
+
     director = StringField("Director", validators=[Required()])
+    director_phone = StringField("Director's Phone", validators=[Required()])
+    director_email = StringField("Director's Email", validators=[Required()])
+    director_address = StringField("Director's Address (Optional)")
+
+    sponsor = StringField("Tournament Sponser (Optional)")
+    sponsor_email = StringField("Sponsor's Email (Optional)")
+    sponsor_phone = StringField("Sponsor's Phone (Optional)")
+    sponsor_website  = StringField("Sponsor's Website (Optional)")
+    sponsor_address = StringField("Sponsor's Address (Optional)")
+
+    convener = StringField("Tournament Convener (Optional)")
+    convener_email = StringField("Convener's Email (Optional)")
+    convener_phone = StringField("Convener's Phone (Optional)")
+    convener_website  = StringField("Convener's Website (Optional)")
+    convener_address = StringField("Convener's Address (Optional)")
+
     pairing = StringField("Pairing", validators=[Required()])
```

Comment:
Here's the XML from the sample: 
`<PairingFormat>McMahon</PairingFormat>`
So, I'm imagining it's either a string or, if it's an enum field of some sort, it points to a table containing valid types of pairings.